nucypher/cli.py

```diff
@@ -78,16 +80,23 @@ class NucypherClickConfig:
     _KEYRING_PASSPHRASE_ENVVAR = "NUCYPHER_KEYRING_PASSPHRASE"
 
     # Set to False to completely opt-out of sentry reporting
-    log_to_sentry = os.environ.get(__LOG_TO_SENTRY_ENVVAR, True)
+    log_to_sentry = True  # TODO: Use envvar
+    log_to_file = True    # TODO: Use envvar
 
     # Pending Configuration Named Tuple
     PendingConfigurationDetails = collections.namedtuple('PendingConfigurationDetails',
                                                          'passphrase wallet signing tls skip_keys save_file')
 
     def __init__(self):
+
+        #
+        # Logging
```

Comment:
Is this how we're doing docstrings now?

Reply:
Haha - no, this is not intended to be a docstring.